[2024-08-16T11:59:35.737+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-16T11:59:35.836+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_to_snowflake.load_data_clientes scheduled__2024-08-15T00:00:00+00:00 [queued]>
[2024-08-16T11:59:35.870+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_to_snowflake.load_data_clientes scheduled__2024-08-15T00:00:00+00:00 [queued]>
[2024-08-16T11:59:35.875+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-16T11:59:35.956+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_data_clientes> on 2024-08-15 00:00:00+00:00
[2024-08-16T11:59:35.975+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'postgres_to_snowflake', 'load_data_clientes', 'scheduled__2024-08-15T00:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/ingestion.py', '--cfg-path', '/tmp/tmp6h4tyz5x']
[2024-08-16T11:59:35.979+0000] {standard_task_runner.py:91} INFO - Job 99: Subtask load_data_clientes
[2024-08-16T11:59:35.973+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=93) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-16T11:59:35.990+0000] {standard_task_runner.py:64} INFO - Started process 106 to run task
[2024-08-16T11:59:36.148+0000] {task_command.py:426} INFO - Running <TaskInstance: postgres_to_snowflake.load_data_clientes scheduled__2024-08-15T00:00:00+00:00 [running]> on host ff16473fb2e3
[2024-08-16T11:59:36.372+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='postgres_to_snowflake' AIRFLOW_CTX_TASK_ID='load_data_clientes' AIRFLOW_CTX_EXECUTION_DATE='2024-08-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-15T00:00:00+00:00'
[2024-08-16T11:59:36.377+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-16T11:59:36.391+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-08-16T11:59:37.691+0000] {base.py:84} INFO - Using connection ID 'snowflake' for task execution.
[2024-08-16T11:59:37.693+0000] {connection.py:408} INFO - Snowflake Connector for Python Version: 3.11.0, Python Version: 3.12.4, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-08-16T11:59:37.694+0000] {connection.py:1258} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-08-16T12:03:51.359+0000] {connection.py:773} INFO - closed
[2024-08-16T12:03:51.407+0000] {connection.py:779} INFO - No async queries seem to be running, deleting session
[2024-08-16T12:03:51.652+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-16T12:03:51.658+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-16T12:03:51.682+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=postgres_to_snowflake, task_id=load_data_clientes, run_id=scheduled__2024-08-15T00:00:00+00:00, execution_date=20240815T000000, start_date=20240816T115935, end_date=20240816T120351
[2024-08-16T12:03:51.748+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-16T12:03:51.771+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T12:03:51.778+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
